core/src/main/data/scripts/repofilter/default-pattern-repofilter.groovy

```diff
@@ -0,0 +1,25 @@
+package org.commonjava.indy.core.content.group
+
+import org.commonjava.indy.model.core.Group
+
+class DefaultPatternNameGroupRepositoryFilter extends PatternNameGroupRepositoryFilter {
+    @Override
+    boolean canProcess(String path, Group group) {
+        return group.getPackageType().equals("maven")
```

Comment:
I'm thinking if this default one should directly return false, as it does not filter anything now. So this can skip steps of regex matching step.

Reply:
The name is confusing. Actually, we don't use it. Maybe I just remove it.